test-harness/index.ts

```diff
@@ -237,7 +231,9 @@ const createSpec = (specPath) => {
 
             expect(reqRes).toMatchObject(masterFile);
 
-            expect(payload.detail).toContain("should have required property 'name'");
+            expect(payload.detail).toContain(
```

Comment:
This line is less interesting than the previous assertion: checking it is moaning about required proper name. Can we assert that this new validation property has the validation we are interested in instead?

Reply:
I'll fix it, however all this is going to be removed soon/needs to be converted to the new harness thing — is it worth to spend time on this?

Reply:
Ahh yeah sorry i didnt notice it was in there. Ignore.